### PR TITLE
Solved DOM error, added custom tu parse, added claim if kak value high

### PR DESCRIPTION
**Pull request by argoniseur**
Solved the DOM error by performing the actions inside the try #2 
Added a claim attempt on character with 300+ kak value, only if they're not in the like/lovelist (meaning there already had an attempt)
Added custom $tu parsing by cutting the big regex in several little ones (might still be errors I missed) #3 
Made the bot roll on launch if it has rolls available

### CODE REVIEW

**Comment by argoniseur** [comment]
#2 #3

File: classes/timers.py [hunk 1]
```diff
@@ -26,7 +28,7 @@ class Timer:
         Whether a kakera loot is currently available or not.
     """
 
-    def __init__(self, browser, next_claim, next_roll, next_daily, claim_available, next_kakera, kakera_available):
+    def __init__(self, browser, next_claim, next_roll, next_daily, claim_available, next_kakera, kakera_available, rollsAtLaunch):
```
Comment (ANTFOR7717) [inline classes/timers.py:31]: **Best practice** is to follow project naming conventions which is _snake_case_ in this scenario:

refactor: `rollsAtLaunch` => `rolls_at_launch`

File: classes/timers.py [hunk 2]
```diff
@@ -44,6 +46,7 @@ def __init__(self, browser, next_claim, next_roll, next_daily, claim_available,
         self.logger.info('Timer created')
         self.logger.info(f'Claim is {"available" if claim_available else "unavailable"}')
         self.logger.info(f'Kakera loot is {"available" if kakera_available else "unavailable"}')
+        self.roll_count = rollsAtLaunch
```
Comment (ANTFOR7717) [inline classes/timers.py:49]: refactor

**Review by ANTFOR7717 — CHANGES_REQUESTED** [review]
This is an excellent addition, I only ask that you push the changes refactored to snake_case

**Comment by ANTFOR7717** [comment]
Looks Good!

**Pull request merged**